Sources/ApolloCodegenLib/UntypedGraphQLRequestBodyCreator.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+
+class UntypedGraphQLRequestBodyCreator {
```

Comment:
If this is only being used for static functions and is never going to be instantiated, the best way to do that is to make it an enum with no cases. (enum or struct are better for performance than classes also)

I feel like we shouldn't be duplicating this code in both targets. If it ever needs to change for bug fixes, changes should be done in one place. Can we move this object into `ApolloUtils`, then have the "typed" `RequestBodyCreator` in `Apollo` call into this function? Like, if it has the operation, it can just pass all the properties from the operation into this function and it should work just right still, yeah?

Reply:
...maybe. I know I tried something like this like two months ago and for some reason it wasn't working, and now i can't remember why

Reply:
I had a crack at this and after talking with @AnthonyMDev we decided it was not worth it; maybe the same hassles you experienced @designatednerd.

_If you're interested to see what changes were needed, you can [take a look at the diff](https://gist.github.com/calvincestari/b5066f59079762f326b93ec125794e1c)._
* _Having to use mapValue ([here](https://gist.github.com/calvincestari/b5066f59079762f326b93ec125794e1c#file-untypedgraphqlrequestbodycreator-diff-L127) and [here](https://gist.github.com/calvincestari/b5066f59079762f326b93ec125794e1c#file-untypedgraphqlrequestbodycreator-diff-L132)) is not great, particularly since this would be called for each typed request too_
* _I also didn't like having to expose `UntypedGraphQLRequestBodyCreator` as `public`_